data/preset/A.yaml

```diff
@@ -37,6 +35,10 @@ defines:
           shortdef: 'Arrangement.Overflow(orientation: horizontal)'
           slots:
             card: *card
+            properties:
```

Comment:
you need to add a type in order to add properties
```
card:
  type: *card
  properties:
```
Anyways I pushed a fix for so you can take a look at it and squash it with this commit